src/mds/Server.cc

```diff
@@ -5546,6 +5546,19 @@ void Server::handle_client_setdirlayout(MDRequestRef& mdr)
 }
 
 // XATTRS
+int Server::check_if_pool_is_empty(const OSDMap& osdmap, int64_t pool_id)
```

Comment:
Is it better to rename the function as `check_if_pool_has_snaps` instead ?

Reply:
renamed function to `check_for_pool_snaps()`